From bb19491cf5e3b78f2d72bbcbcc06e985d29b7f4b Mon Sep 17 00:00:00 2001
From: Peter Krempa <pkrempa@redhat.com>
Date: Fri, 11 Jan 2013 11:10:34 +0100
Subject: [PATCH] network: bridge: Fix regression when defining persistent
 networks

Commit 0211fd6e04cdc402da20818df54299c6ded3d3cb introduced regression
where newly defined networks were not made persistent.

This patch makes the network persistent on each successful definition.
---
 src/network/bridge_driver.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/network/bridge_driver.c b/src/network/bridge_driver.c
index 660c38d..975b7f6 100644
--- a/src/network/bridge_driver.c
+++ b/src/network/bridge_driver.c
@@ -3124,6 +3124,9 @@ static virNetworkPtr networkDefine(virConnectPtr conn, const char *xml) {
             goto cleanup;
     }
 
+    /* define makes the network persistent - always */
+    network->persistent = 1;
+
     /* def was asigned */
     freeDef = false;
 
-- 
1.8.1

